sampler_test.go

```diff
@@ -387,7 +369,7 @@ func TestUpdateSampler(t *testing.T) {
 		{
 			map[string]float64{testOperationName: 1.1},
 			testDefaultSamplingProbability,
-			"|phase=parsing|state=failure",
+			"|phase=updating|state=failure",
 			1,
 			true,
 		},
```

Comment:
we really should be using named params instead of positional. I do remember getting a lint or vet errors about it elsewhere.